src/OrchardCore/OrchardCore/Environment/Shell/Configuration/ShellsConfigurationSources.cs

```diff
@@ -19,6 +19,7 @@ public ShellsConfigurationSources(IHostEnvironment hostingEnvironment, IOptions<
         public void AddSources(IConfigurationBuilder builder)
         {
             builder
+                .AddEnvironmentVariables()
```

Comment:
@sebastienros maybe at some point on setup

But not now because we use `CreateDefaultSettings()` that takes into account all the app level config sources.

Reply:
Including the env variables, so it is not necessary to use `AddEnvironmentVariables` here